README.md

```diff
@@ -1,4 +1,5 @@
-# MongoDB Cookbook
+# MongoDB Cookbook for by JMC
+Note: this fork has been created to create pull request in chef-brigade Repo. Do not use directly when PR are integrated.
```

Comment:
Yes of course. Can you merge all but rhe README ?
Le 6 oct. 2015 3:03 PM, "Thomas Riccardi" <notifications@github.com> a
écrit :

> In README.md
> <https://github.com/chef-brigade/mongodb-cookbook/pull/94#discussion_r41260145>
> :
>
> > @@ -1,4 +1,5 @@
> > -# MongoDB Cookbook
> > +# MongoDB Cookbook for by JMC
> > +Note: this fork has been created to create pull request in chef-brigade Repo. Do not use directly when PR are integrated.
>
> cannot merge this
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chef-brigade/mongodb-cookbook/pull/94/files#r41260145>
> .
>